src/services/simulationService.js

```diff
@@ -62,27 +62,35 @@ export class SimulationService {
       .catch(resolveConflict);
   }
 
-  /** Clones an existing simulation */
-  static cloneSimulation(model) {
-    return HttpClient.post(
-        `${ENDPOINT}simulations`,
-        toSimulationCloneModel(model)
-      )
-      .map(toSimulationModel)
-      .catch(resolveConflict);
+    /** Start/Restart a simulation */
+  static startSimulation(simulation) {
+    return SimulationService.getSimulation(simulation.id)
+      .flatMap (({ eTag }) => {
+        return HttpClient.put (
+            `${ENDPOINT}simulations/${simulation.id}`,
+            toSimulationUpdateModel({ ...simulation, ETag: eTag })
+          )
+          .map (toSimulationModel)
+          .catch (resolveConflict);
+      });
   }
-
   /** Disable a simulation */
   static stopSimulation(simulation) {
     return SimulationService.getSimulation(simulation.id)
-      .flatMap(({ eTag }) => {
-        return HttpClient.patch(
+      .flatMap (({ eTag }) => {
```

Comment:
Please remove extra spaces.
```suggestion
      .flatMap(({ eTag }) => {
```